javascript/house-style.md

```diff
@@ -343,51 +342,33 @@ We use automated tools (like Snyk) to monitor the status of dependencies in our
 
 We use white space liberally to help keep code readable. You're encouraged to use newlines to break up long functions into logical chunks.
 
-You should also remove trailing white space from lines of code. Your editor should be able to either remove this or highlight it when present, for example, [atom-whitespace](https://github.com/atom/whitespace).
+You should also remove trailing white space from lines of code. Your editor should be able to either remove this or highlight it when present, for example, [atom-whitespace](https://github.com/atom/whitespace), [vscode-trailing-spaces](https://marketplace.visualstudio.com/items?itemName=shardulm94.trailing-spaces).
 
-### Semi-colons
+### Variables
 
-We require the use of semi-colons. The only statements that don't need to end with semi-colons are function declarations and blocks.
+#### Use of const and let
+We define variables with multiple `const` and `let` statements.
 
 We do this:
 
 ```js
-var foo = 1;
-
-if (foo) {
-    foo += 1;
-}
-
-function bar() {
-    // ...
-}
-
-var baz = function() {
-    // ...
-};
+const foo = 1;
+const bar = 2;
 ```
 
 We _don't_ do this:
 
 ```js
-var foo = 1
-
-if (foo) {
-    foo += 1
-};
-
-function bar() {
-    // ...
-};
-
-var baz = function() {
-    // ...
-}
+const foo = 1,
+      bar = 2;
 ```
 
-### Variables
+We use `let` only when a variable _explicitly_ [needs to be mutable](https://ada.is/blog/2015/07/13/immutable/):
+
+#### Use of var
 
-We define variables with multiple `var` statements and trust our developers to understand [hoisting](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/var#var_hoisting). We generally discourage defining variables inside loops or blocks.
+If the environment you're running in does not support ES2015 (Node.js 4.x/[Babel](https://babeljs.io/)), and therefore you are using `var` instead of `const` and `let`, we trust our developers to understand [hoisting](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/var#var_hoisting). 
```

Comment:
This sentence is a bit tortured. I know it's not your original work though, so you are blameless 😄 

```suggestion
If you have to use `var` instead of `const` and `let` (because your environment doesn't support ES2015 (Node.js 4.x/[Babel](https://babeljs.io/))), you need to understand [hoisting](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/var#var_hoisting). 
```